studio/components/layouts/ProjectLayout/UpgradingState/UpgradingState.tsx

```diff
@@ -177,7 +178,7 @@ const UpgradingState = () => {
                                 />
                               </div>
                             ) : isCompleted ? (
-                              <div className="flex items-center justify-center w-5 h-5 border rounded-full bg-brand-300 border-brand-400">
+                              <div className="flex items-center justify-center w-5 h-5 border rounded-full bg-brand-600 dark:bg-brand-300 border-brand-400">
```

Comment:
@ivasilov 

these are old css classes that we are deprecating.

Reply:
don't need dark:

Reply:
Hm, this change was done since some visual elements in the upgrade progress page didn't have enough contrast in light mode, though might just be due to the use of old classes

![53172](https://github.com/supabase/supabase/assets/30039543/ed198dd5-1426-43af-bc75-0e1600711554)

Reply:
ahh, we use a brand scale of 100 - 600 now 

so you probably wanted something like bg-brand-300 border-brand-400

no need to use `:dark` stuff now, they will work in both.

Reply:
love the status list btw @pcnc

Reply:
Totally @joshenlim's brainchild!